src/components/scenes/FioAddressListScene.js

```diff
@@ -92,9 +92,10 @@ class FioAddressList extends React.Component<Props, LocalState> {
 
   onAddressPress = (fioAddress: FioAddress) => {
     const { navigation } = this.props
-    const { name } = fioAddress
+    const { name, bundles } = fioAddress
```

Comment:
Please change all variables that have `bundles` to `bundledTxs` or `bundled_txs` bundles is not a proper name